src/pybind/mgr/cephadm/module.py

```diff
@@ -1975,6 +1975,9 @@ def remove_service(self, service_name: str, force: bool = False) -> str:
             if self.spec_store[service_name].spec.service_type in ('mon', 'mgr'):
                 return f'Unable to remove {service_name} service.\n' \
                        f'Note, you might want to mark the {service_name} service as "unmanaged"'
+        else:
+            raise OrchestratorError(f"Invalid service '{service_name}'. Use 'ceph orch ls' " \
```

Comment:
```
cephadm/module.py:1979:94: E502 the backslash is redundant between brackets
1     E502 the backslash is redundant between brackets
```
flake8 error

Reply:
I'll fix it .. but I'm not sure why I don't see the same error on my environment.